### PR TITLE
Cldc 3115 reduce dev db spec

### PR DESCRIPTION
**Pull request by RachaelBooth**
This looks like it will meet our actual usage needs very comfortably - it does assume we are not using too many review apps heavily at once, which feels like a safe assumption.

We may even be able to get away with micro instead of small, but the savings there are much smaller scale so it feels best to start with this.

### CODE REVIEW

**Comment by RachaelBooth** [comment]
N.B. There will be downtime (for all review apps) when applying this change to the dev db.

File: terraform/modules/rds/variables.tf [hunk 1]
```diff
@@ -30,7 +36,8 @@ variable "enable_primary_deletion_protection" {
 
 variable "enable_replica_deletion_protection" {
   type        = bool
-  description = "Whether the replica database should have deletion protection enabled"
+  description = "Whether the replica database (if create_replica is true) should have deletion protection enabled"
+  default     = true
```
Comment (kosiakkatrina) [inline terraform/modules/rds/variables.tf:40]: What is the benefit of setting the default here?

Comment (RachaelBooth) [reply terraform/modules/rds/variables.tf:40]: My reasoning here was it meant not needing to specify a value when there is no replica (which is a weird thing to have to do) - annoyingly you can't get interesting conditions that reference other variables, but true seems like a reasonable default here if you do have a replica and don't specify.

Comment (kosiakkatrina) [reply terraform/modules/rds/variables.tf:40]: Yup, that makes sense

**Review by kosiakkatrina — APPROVED** [review]

**Pull request merged**